pandas/core/internals/blocks.py

```diff
@@ -1318,7 +1318,7 @@ def shift(self, periods, axis=0, mgr=None):
 
         return [self.make_block(new_values)]
 
-    def eval(self, func, other, errors='raise', try_cast=False, mgr=None):
+    def eval(self, func, other, errors='raise', mgr=None):
```

Comment:
Given recent implementation of ops.dispatch_to_series, we only ever get here with `try_cast=False`, so we can remove the kwarg altogether.